src/Microsoft.Identity.Web/CertificateManagement/DefaultCertificateLoader.cs

```diff
@@ -27,19 +27,19 @@ public void LoadIfNeeded(CertificateDescription certificateDescription)
                 switch (certificateDescription.SourceType)
                 {
                     case CertificateSource.KeyVault:
-                        certificateDescription.Certificate = LoadFromKeyVault(certificateDescription.Container, certificateDescription.ReferenceOrValue);
+                        certificateDescription.Certificate = LoadFromKeyVault(certificateDescription.Container!, certificateDescription.ReferenceOrValue!);
```

Comment:
`!` overrides the conversion from nullable to non-nullable type, which otherwise would throw an exception. In this case, the properties shouldn't be null.